### ISSUE TITLE
[Enhancement] Better Information about leftovers

### ISSUE THREAD

**Issue by Leinad4Mind** [AlexSheer/phpBB3.2-STK#37]

**Comment by AlexSheer** [comment]
You can check which extension the item belongs to by clicking the Extra link.
It is impossible to check which mode the elements belong to.
![Screenshot_1](https://user-images.githubusercontent.com/9640205/62208643-29dd0280-b3a0-11e9-810f-827e52db057b.png)
![Screenshot_2](https://user-images.githubusercontent.com/9640205/62208652-2e092000-b3a0-11e9-8d42-49e4b419a418.png)

**Comment by Leinad4Mind** [comment]
😲 Awesome, Thanks for the info! 

About mods, what do you think on creating a file for that, and people would send PRs to make it grow bigger, so we would with time have a database with all the mods infos. You could make the 1st object with 1 Mod example and peoople will follow the logic and add more to it.

Cheers 🚀

**Comment by AlexSheer** [comment]
I do not think this is real. Well, maybe only for the most popular mods, such as **Thanks for post (Rating edition)**

**Comment by Leinad4Mind** [comment]
I used back in the days almost all best mods, as mchat, bbportal, prime mods, its a good list of them.
Since I'll go trhough all of them to find where a phpbb_countries come from, I could use my time to populate this list too.
